### ISSUE TITLE
Bug: Flash mods info popup not clearing itself

### ISSUE THREAD

**Issue by Ashfaaq18** [marcelstoer/nodemcu-pyflasher#80]
As shown in the picture below, when I hover over the flash mods info icon, the info menu pops up. 
Once the cursor moves away from the info icon, the info menu does not clear itself. 
Even moving or minimizing the window does not clear the info menu.

![image](https://user-images.githubusercontent.com/27722888/134775100-11561040-b91b-40e9-93cd-b7e62e23eee0.png)

My win10 specs:
version: 20H2
OS build: 19042.1237

**Comment by marcelstoer** [comment]
On my system I can click the overlay (or even the main window) to make it disappear.

**Comment by Ashfaaq18** [comment]
I'm assuming you tested under windows. If yes, then yeah, kinda strange. 
I tried restarting my pc and running the app again. Still the same issue. Not sure if a windows update messed this up.

**Comment by marcelstoer** [comment]
I can confirm this to be an issue with PyFlasher v5 on a recent Windows 10 system.

**Comment by chall3ng3r** [comment]
Yes, latest build have this issue on Windows 11 as well. Popup does not disappear once open.

**Comment by 0xenca** [comment]
Confirmed Windows 10 still 2022
![Sin título](https://user-images.githubusercontent.com/106509878/181363603-d307ae17-7ae6-4598-be4a-a68ea3c704df.png)

**Comment by JonnyPhenomenon** [comment]
HAH! yeah, that pop over is annoying. I've got three of them on my screen as I type this.  plus the three in the screen shot above! lol!

**Comment by marcelstoer** [comment]
Actually appears to be an incarnation of issue https://github.com/wxWidgets/Phoenix/issues/2554.

**Comment by marcelstoer** [comment]
v5.1 should fix this, please verify, thanks.

https://github.com/marcelstoer/nodemcu-pyflasher/releases/tag/v5.1.0
